src/utils/common.js

```diff
@@ -10,3 +11,30 @@ export async function randomHex (nBytes = 16) {
 	const bytes = await random(nBytes)
 	return bytes.toString('hex')
 }
+
+export async function createAuthUrl(schoolUrl) {
```

Comment:
I think that case never occurs.

Reply:
I think you may've responded to the wrong comment? 
https://github.com/simplyGits/MagisterJS/pull/143/files#r302710346

Also I do agree that common.js isn't totally the right place for this function. Give it it's own file inside /utils or don't put it seperate at all : )

Reply:
I think I agree this could maybe be moved somewhere else. Don't really feel like this should be an util function. I think it would be actually fine to have this as a lambda in the login function or something like that?